src/client/sections/device-section.tsx

```diff
@@ -44,12 +44,17 @@ export function DeviceSection() {
           className='flex flex-wrap gap-6 p-4 items-stretch content-start
           rounded-lg border border-dashed shadow-sm relative'
         >
-          {_.some(device.capabilities, (capability: Capability) =>
-            capability.id === 'remote-teleop') && (
+          {device.capabilities['robot-lock'] && (
+              <div className='w-full'>
+                <JWTCapability device={deviceId}
+                  capability={device.capabilities['robot-lock'].scope + '/robot-lock'} />
+              </div>
+          )}
+          {device.capabilities['remote-teleop'] && (
               <div className='grow-0 basis-3/5 shrink m-auto'>
                 <JWTCapability
                   device={deviceId}
-                  capability={'@transitive-robotics/remote-teleop'}
+                  capability={device.capabilities['remote-teleop'].scope + '/remote-teleop'}
```

Comment:
Same as above. Shouldn't we simply have the scope in the fleet context and then pass it down, rather than rediscovering, looking it up, it every time?